src/jdk.compiler/share/classes/com/sun/tools/javac/resources/launcher.properties

```diff
@@ -110,7 +110,11 @@ launcher.err.cant.find.class=\
 
 # 0: string
 launcher.err.cant.find.main.method=\
-    can''t find main(String[]) method in class: {0}
+    can''t find main(String[]) or main() method in class: {0}
```

Comment:
Out of curiosity, why the double single quotes? I see it being used in other messages in this file as well, so it doesn't seem to be a typo.

Reply:
Pieces of text in `MessageFormat` can be quoted using `'...'`. So, a single `'` does not work, as that is considered to be part of quoting. Using `''` escapes that behavior.